builder/install/70-valijson.sh

```diff
@@ -9,7 +9,6 @@ mkdir cmake-build && cd cmake-build
 cmake \
     -DCMAKE_BUILD_TYPE=Release \
     -DCMAKE_INSTALL_PREFIX=/usr/local \
-    -Dvalijson_INSTALL_HEADERS=ON \
```

Comment:
Falco doesn't need valijson headers anymore?

Reply:
No, the newer version of valijson got rid of this option, so I just removed it, it will install the headers by default.